### PR TITLE
Refactor external strings

### PR DESCRIPTION
**Pull request by dainnilsson**
This PR reorganizes the external strings used by the app, as well as extracts most of the remaining hard-coded values.
Strings have been renamed, deduplicated, and reordered.

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -1,301 +1,430 @@
...
+    "l_insert_or_tap_yk": "Insert or tap a YubiKey",
+    "l_unplug_yk": "Unplug your YubiKey",
+    "l_reinsert_yk": "Reinsert your YubiKey",
+    "l_place_on_nfc_reader": "Place your YubiKey on the NFC reader",
+    "l_replace_yk_on_reader": "Place your YubiKey back on the reader",
+    "l_remove_yk_from_reader": "Remove your YubiKey from the NFC reader",
+    "l_try_reinsert_yk": "Try to remove and reinsert your YubiKey.",
+    "l_touch_required": "Touch Required",
+    "l_touch_button_now": "Touch your YubiKey now",
+    "l_touch_button_now": "Touch the button on your YubiKey now",
```
Comment (AdamVe) [inline lib/l10n/app_en.arb:105]: it looks like this key is twice here

Comment (AdamVe) [reply lib/l10n/app_en.arb:105]: On Android we originally used `Touch the button on your YubiKey now.`

Comment (dainnilsson) [reply lib/l10n/app_en.arb:105]: Good catch! We can probably choose one and use it for both platforms.

File: lib/l10n/app_en.arb [hunk 2]
```diff
@@ -1,301 +1,430 @@
...
-    "widgets_close": "Close",
+    "@_yubikey_interactions": {},
+    "l_insert_yk": "Insert your YubiKey",
+    "l_insert_or_tap_yk": "Insert or tap a YubiKey",
+    "l_unplug_yk": "Unplug your YubiKey",
+    "l_reinsert_yk": "Reinsert your YubiKey",
+    "l_place_on_nfc_reader": "Place your YubiKey on the NFC reader",
+    "l_replace_yk_on_reader": "Place your YubiKey back on the reader",
+    "l_remove_yk_from_reader": "Remove your YubiKey from the NFC reader",
+    "l_try_reinsert_yk": "Try to remove and reinsert your YubiKey.",
```
Comment (AdamVe) [inline lib/l10n/app_en.arb:102]: note: in original it is spelled re-insert, but reinsert is correct.

File: lib/l10n/app_en.arb [hunk 3]
```diff
@@ -1,301 +1,430 @@
...
+    "l_add_fingerprint": "Add fingerprint",
+    "l_fp_step_1_capture": "Step 1/2: Capture fingerprint",
+    "l_fp_step_2_name": "Step 2/2: Name fingerprint",
+    "l_delete_fingerprint": "Delete fingerprint",
+    "l_fingerprint_deleted": "Fingerprint deleted",
+    "l_warning_delete_fingerprint": "This will delete the fingerprint from your YubiKey.",
+    "l_no_fingerprints": "No fingerprints",
+    "l_set_pin_fingerprints": "Set a PIN to register fingerprints",
+    "l_no_fps_added": "No fingerprints have been added",
+    "l_fingerprint_renamed": "Fingerprint renamed",
```
Comment (AdamVe) [inline lib/l10n/app_en.arb:313]: the naming of the keys for `fingerprint` sometimes uses `fp`/`fps`. I think it would be nice to unify this, so one can easily find all fingerprint related UI strings by search for a one string (either use only fp or only fingerprint)

File: lib/oath/keys.dart [hunk 4]
```diff
@@ -18,7 +18,7 @@ import 'package:flutter/material.dart';
 
 const _prefix = 'oath.keys';
 
-const setOrManagePasswordAction = Key('$_prefix.set_or_manage_oath_password');
+const setOrManagePasswordAction = Key('$_prefix.set_or_manage_w_password');
```
Comment (AdamVe) [inline lib/oath/keys.dart:21]: is this change intended? should be better with `set_or_manage_password`. note that this is not a localization key but a string for Widget key.

**Review by AdamVe — APPROVED** [review]
👍🏻
